packages.json

```diff
@@ -14788,5 +14788,23 @@
     "description": "256 colors for shell",
     "license": "MIT",
     "web": "https://github.com/thebigbaron/rainbow"
+  },
+  {
+    "name": "rtree",
+    "url": "https://github.com/stefansalewski/RTree",
+    "method": "git",
+    "tags": [
+      "RTree",
+      "R-Tree",
+      "R*Tree",
+      "R*-Tree",
```

Comment:
Please remove these. These aren't good tags, they are just variations of the name. Tags should reflect some overarching categories like `library` or `wrapper` or `network` etc.